test_integration/shared.ts

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+import {Axiom} from "../interacts/Axiom";
+import {Committee} from "../interacts/Committee";
+import {Consensus} from "../interacts/Consensus";
+import {Core} from "../interacts/Core";
+import {Reputation} from "../interacts/Reputation";
+import {ERC20I} from "../interacts/ERC20I";
+import {Anchor} from "../interacts/Anchor";
+
+const Web3 = require("web3");
+
+const web3 = new Web3('http://localhost:8545');
```

Comment:
Suggestion: Read this value from a constant file or env.
No need to change this now, its just a note.